pipelines/resources/edgeclusters/edgecluster-deploy-lso.yaml

```diff
@@ -36,11 +48,8 @@ spec:
         #!/usr/bin/bash
 
         if [[ "${MOCK}" == 'false' ]]; then
-          ztp create lso \
-          --config "$(params.edgeclusters-config)" \
-          --log-field "pr=$(params.pipeline-name)" \
-          --log-field "tr=$(context.taskRun.name)" \
-          $(params.ztp-extra-args)
+          cd ${WORKDIR}/${DEPLOY_ODF_DIR}
+          ./deploy.sh
         else
-          echo "Deploy LSO: Mock mode on"
+          echo "Deploy ODF: Mock mode on"
```

Comment:
The name of the step and the yaml has "lso"

Reply:
Yes. I fixed that comment in one of the changes I did. Now is reverted and incorrect again. I'd prefer to keep it like this just to do a clean revert. If you think it needs to be changed I will do it in a separate patch.